src/Core/Domain/Category/SeoSettings.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Domain\Category;
+
+/**
+ * Defines settings for Category's SEO
+ */
+class SeoSettings
+{
+    /**
+     * Maximum length of SEO title
+     */
+    const MAX_TITLE_LENGTH = 70;
+
+    /**
+     * Maximum length of SEO description
+     */
+    const MAX_DESCRIPTION_LENGTH = 160;
```

Comment:
This is something we discussed with @colinegin and @jolelievre, We mustn't force people to have a 70 chars title & 160 for description. It must be a recommandation, not a requirement!

Reply:
:confused: 

What do you mean? It does not force, it's just a max allowed value? You can have with 0, 10, 100 or 160 (max) characters. Or am I wrong?

Reply:
But you can't have 200 characters or 80 for the title

Reply:
Yeah, but we must limit it to atleast 255, as it hits database column limit, right?

Reply:
Yep!

Reply:
Or if it's a Text / Blob type, more (like 512)